roles/vendor/redis/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+#Redis version
+redis_stable_version: '7.0.4'
```

Comment:
the whole section is not actually anymore, since we want to install the latest from PPA?